index.d.ts

```diff
@@ -1,3 +1,12 @@
+export interface Options {
+	/**
+	Count [Ambiguous characters](https://www.unicode.org/reports/tr11/#Ambiguous) as a width of 1, false counts as 2.
+
+	@default false
+	*/
+	readonly ambiguousIsNarrow: boolean;
```

Comment:
You need to document it in the readme too. Using this formatting: https://github.com/sindresorhus/boxen#api